Cargo.toml

```diff
@@ -10,3 +10,6 @@ members = [
   "neqo-interop",
   "test-fixture",
 ]
+
+[patch.crates-io]
+qlog = { git = "https://github.com/mxinden/quiche", branch = "type-to-importance" }
```

Comment:
To be removed once https://github.com/cloudflare/quiche/pull/1684 is merged and released.

Reply:
We'll want to wait for that before we merge here.  For future reference, a draft PR is probably better for cases where you need to wait for upstream changes.  That might delay review, but we're generally happy to provide feedback on direction from a draft.

Reply:
https://github.com/cloudflare/quiche/pull/1684 is merged.

Reply:
@mxinden could you update the PR to not require the patched crate?

Reply:
I am sorry for the delay here @larseggert.

With https://github.com/cloudflare/quiche/pull/1684 merged, I updated from `github.com/mxinden` to `github.com/cloudflare`. See https://github.com/mozilla/neqo/pull/1505/commits/5884b456e10c5e1f6a52e89af8c0fcacc86f07dd. I am assuming that we still want to block merging here until the patch in https://github.com/cloudflare/quiche/pull/1684 has been released to crates.io. See also https://crates.io/crates/qlog/versions. Let me know in case I am missing something.

Reply:
Yes, thanks for catching that. (I had assumed quiche would be uploaded to crates.io after the merge, but I guess not yet.)

Reply:
Patch section removed with https://github.com/mozilla/neqo/pull/1505/commits/4754645b1c6192fdf6a5d17a927acae2da18c17a with upgrade to `qlog` `v0.11.0` in https://github.com/mozilla/neqo/pull/1547. Resolving here.